charts/metallb/README.md

```diff
@@ -1,6 +1,6 @@
 # metallb
 
-![Version: 0.9.6](https://img.shields.io/badge/Version-0.9.6-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.9.6](https://img.shields.io/badge/AppVersion-v0.9.6-informational?style=flat-square)
+![Version: 0.12.1](https://img.shields.io/badge/Version-0.12.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.12.1](https://img.shields.io/badge/AppVersion-v0.12.1-informational?style=flat-square)
```

Comment:
Same as in #1369, it looks like there are missing fields in the `README.md` that are getting pulled in here.  @fedepaol  we may need to look at the release process and possibly automate the chart `README.md` generation

Reply:
Yeah automating and enforcing would be great.
We are doing it already for a bunch of other resources, CI won't allow the PR in if the content is not aligned.

Reply:
Filed https://github.com/metallb/metallb/issues/1373 to track this